etc/config/c++.amazon.properties

```diff
@@ -1,4 +1,4 @@
-compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&clang-rocm:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502:&nvcxx_x86_cxx:&nvcxx_arm_cxx
+compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&clang-rocm:mosclang:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502:&nvcxx_x86_cxx:&nvcxx_arm_cxx
```

Comment:
Can you make this a group like the others? This way it will be easier to change & add its properties in the future

Reply:
Done.